client-app/src/pages/account/views/AccountOrders.vue

```diff
@@ -69,6 +71,42 @@ export default class AccountOrders extends Vue{
 
   mounted() {
     this.fetchOrders();
+    this.resizeTableColumns();
+  }
+
+  resizeTableColumns() {
```

Comment:
I suggest moving this code from smart-component logic into a directive https://vuejs.org/v2/guide/custom-directive.html.